content/en/news/releases/1.17.x/announcing-1.17/_index.md

```diff
@@ -0,0 +1,75 @@
+
+### Enhancements to Istio dual-stack feature
+
+Added support  to generate configuration for Istio gateway with dual stack enabled. `IPv6` support in dual-stack mode was added in Kubernetes in version 1.16 and graduated to stable in the 1.22 release. Istio dual stack support is an alpha feature and was introduced in release 1.16. This is an area under [active development]( https://github.com/istio/istio/issues/40394).
+
+### Added support for filter patching in Istio
+
+Added support for listener filter patching which enables users to perform `ADD`, `REMOVE`, `REPLACE`, `INSERT_FIRST`, `INSERT_BEFORE`, `INSERT_AFTER` operations for `LISTENER_FILTER` in `EnvoyFilter`. 
+
+### Added support for using `QuickAssist Technology` (QAT) `PrivateKeyProvider` in Istio
+
+Added support for using `QuickAssist Technology` (QAT) `PrivateKeyProvider` in SDS and added corresponding configuration for selecting QAT private key provider for gateways and sidecars. This builds on the fact that Envoy added [support for QAT]( https://github.com/envoyproxy/envoy/issues/21531) as another private key provider in addition to [CryptoMB]( https://istio.io/latest/blog/2022/cryptomb-privatekeyprovider/). For more information on QAT, you can refer [here]( https://www.intel.com/content/www/us/en/developer/articles/technical/envoy-tls-acceleration-with-quickassist-technology.html).
+
+### Enhancements to the `RequestAuth` API
+
+Added support to copy JWT claims to HTTP request headers in the `RequestAuth` API.
+
+### Enhancements to the `istioctl` command
+
+Added a number of enhancements to the istioctl command including adding:
+
+- `revision` flag to admin log, to switch controls between Istiod’s
+- `istioctl proxy-config ecds`, to support retrieving typed extension configuration from Envoy for a specified pod
+- `istioctl proxy-config log`, to set proxy log level for all pods in a deployment
+- adding the `--revision` flag to `istioctl analyze` to specify a specific revision.
+
+## Join us at Istio Day, 2023
+
+[Istio Day 2023](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/istio-day/), set for April 18th, will be the first annual conference for the Istio community. Istio Day will be a Day 0 event co-located with [KubeCon + CloudNativeCon Europe 2023](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe), connecting community members across the globe with Istio’s ecosystem of developers, partners and vendors. For more information related to the event, visit the [conference website](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/). We look forward to seeing you at Istio Day in April.
```

Comment:
Wasn't IstioCon an annual conference and would be the first? Maybe IstioDay is the first in-person?

Reply:
Rephrased the sentence.

Reply:
Should be the first hosted by CNCF

Reply:
Talked with @johnma14 on this and sounds like this is resolved now.